packages/atomic/src/components/atomic-search-interface/analytics-config.ts

```diff
@@ -22,7 +22,6 @@ export function getAnalyticsConfig(
   const defaultConfiguration: AnalyticsConfiguration = {
     analyticsClientMiddleware,
     enabled,
-    documentLocation: document.location.href,
```

Comment:
Why is this removed?

Reply:
oh, thanks for catching that!! it wasn't supposed to be committed.. I think it was something to do with a bad merge from master and then the linter complaining about this. I'll go through the whole code one more time to make sure there is nothing extra